### ISSUE TITLE
Missing close button on some Android phones

### ISSUE THREAD

**Issue by rfmejia** [OneSignal/react-native-onesignal#303]
On certain Android devices, the push notification dialog does not show the default "close" action button, thus requiring the user to use the OS back button to close the dialog. This is observed on notifications sent using both the web console and REST API.

We are using react-native-onesignal v3.0.3, and our (limited) set of devices where the button does not show up include:

- HUAWEI P9 lite, Android 7.0
- Sony Xperia SO-02F, Android 4.4.2

**Comment by saalihou** [comment]
Experiencing the same issue, since I did a build this morning.
Here is an image demonstrating it:

![screenshot aug 18 2017 22-47-13](https://user-images.githubusercontent.com/9271500/29480388-89b5d78c-8467-11e7-95d7-7199ee01b84d.png)

Device is Infinix X555, Android 7.0

Edit: It looks like it is introduced by version 3.6.0 of the onesignal android sdk. As a temporary fix, you can force the sdk version to 3.5.8, in `android/app/build.gradle` file, under the dependencies:
```gradle
compile (project(':react-native-onesignal')){
  exclude group: 'com.onesignal', module: 'OneSignal'
}
compile 'com.onesignal:OneSignal:3.5.8@aar'
```

**Comment by avishayil** [comment]
I'm not sure how to handle this. @jkasten2 @Nightsd01

**Comment by Nightsd01** [comment]
@rfmejia This was fixed in the 3.6.2 OneSignal Android SDK. Sorry for the delayed response!
